libs/zzz/formula/src/data/common/anomalyBuildup.ts

```diff
@@ -0,0 +1,17 @@
+import { prod, sum } from '@genshin-optimizer/pando/engine'
+import type { TagMapNodeEntries } from '../util'
+import { enemy, own, ownBuff, percent } from '../util'
+
+const data: TagMapNodeEntries = [
+  // Anomaly buildup
+  ownBuff.formula.anomalyBuildup.add(
```

Comment:
For this and daze formula, we want something more similar to damage, where we use prod accu and then .add each term individually from another named portion, so it'll get printed nicely on UI

https://github.com/frzyc/genshin-optimizer/blob/f9a068edc26713724b30e3fb4964eafdc67f635a/libs/zzz/formula/src/data/common/dmg.ts#L22